build.gradle

```
@@ -54,63 +63,72 @@ repositories {
+    implementation group: 'jakarta.validation', name: 'jakarta.validation-api'
+    implementation group: 'io.projectreactor', name: 'reactor-core', version: '3.4.34'
 
     // Google dependencies
     constraints {
         implementation group: 'com.google.guava', name: 'guava', version: '31.1-jre' // '-jre' for Java 8 or higher
     }
-    implementation platform('com.google.cloud:libraries-bom:26.0.0') // use common bom
-
-    implementation group: 'com.google.auto.value', name: 'auto-value-annotations', version: '1.9'
-    implementation group: 'com.google.cloud', name: 'google-cloud-billing', version: '2.3.0'
-    implementation group: 'com.google.cloud', name: 'google-cloud-core-http', version: '2.8.0'
-    implementation group: 'com.google.cloud', name: 'google-cloud-pubsub', version: '1.120.0'
-    implementation group: 'com.google.cloud', name: 'google-cloud-storage', version: '2.9.2'
-    implementation group: 'com.google.cloud', name: 'google-cloud-iamcredentials', version: '2.17.0'
+    implementation platform('com.google.cloud:libraries-bom:26.28.0') // use common bom
+    implementation group: 'com.google.auto.value', name: 'auto-value-annotations'
+    implementation group: 'com.google.cloud', name: 'google-cloud-pubsub', version: '1.125.13'
+    implementation group: 'com.google.cloud', name: 'google-cloud-iamcredentials', version: '2.31.0'
     annotationProcessor group: 'com.google.auto.value', name: 'auto-value', version: '1.9'
 
     // Database
-    implementation group: 'org.liquibase' , name: 'liquibase-core', version: '4.17.2'
+    implementation group: 'org.liquibase' , name: 'liquibase-core', version: '4.22.0'
     runtimeOnly group: 'org.postgresql', name: 'postgresql', version: '42.5.0'
 
     // Terra libraries
-    implementation group: 'bio.terra', name: 'terra-common-lib', version: '0.0.89-SNAPSHOT'
-    implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: '1.2.27-SNAPSHOT'
-    implementation group: 'bio.terra', name: 'workspace-manager-client', version: '0.254.575-SNAPSHOT'
+    implementation group: 'bio.terra', name: 'terra-common-lib', version: '0.1.9-SNAPSHOT'
+    implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: '1.2.30-SNAPSHOT'
+    implementation group: 'bio.terra', name: 'workspace-manager-client', version: '0.254.997-SNAPSHOT'
 
     // hk2 is required to use WSM client, but not correctly exposed by the client
-    implementation group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: jerseyVersion
+    implementation group: 'org.glassfish.jersey.inject', name: 'jersey-hk2'
 
     // Kubernetes client
     implementation group: 'io.kubernetes', name: 'client-java', version: '18.0.0'
 
-    // Metrics & Tracing
-    implementation group: 'io.opencensus', name: 'opencensus-exporter-stats-stackdriver', version: '0.31.1'
+    // OpenTelemetry
+    var openTelemetryVersion = '1.31.0'
```

```suggestion
    var openTelemetryVersion = '1.32.0'
```

(As an aside, I'm wondering whether these imports should be included by default in TCL.)

---

Hm yeah, it seems TCL does pull them in.. maybe they are not needed here? Will try.

---

Looks like they do need to be in Janitor because they are marked `implementation` in TCL (so don't get pulled in transitively). I bumped to 1.32.0 though.